### ISSUE TITLE
Allow percentage chance of going to different dimensions after falling into the void

### ISSUE THREAD

**Issue by rebarakuk** [The-Fireplace-Minecraft-Mods/Unforgiving-Void#19]
### Description

I want these to be added

+ In config, I want option that add dimensions where player goes to after falling to void. Also, there should be percentage to go to.

(If the sum of every percentage of dimensions doesn't reach 100%, the rest of percentage is the percentage of dying)

+ Config option that gives player resistance level 255 for some seconds not to die instantly after landing to another dimension. Or an option about what potion effects should player get after landing a dimension and how long should it last

(ex: after landing on Hydra boss of twilight forest, player must die instantly without armor)

**Comment by The-Fireplace** [comment]
The option to control which dimension the player lands in already exists in versions newer than MC 1.12.2. Though it doesn't have a way to set a chance of dying or different chances of going to different dimensions from the same dimension's void

**Comment by rebarakuk** [comment]
sry I was 1.12.2

**Comment by Normal69** [comment]
Yes, just got to see the config after installing it in my old 1.12.2 pack.
The main thing was to assimilate anything interesting from Last Days of Humanity pack. :)

While I have no problems with falling into the Nether (totally rewamped it into an agressive adventure dimension),
but I want to be extra merciful and rather drop the players into the more forgiving Deep D
Dark (#-11325). :3
